include/mbgl/util/projection.hpp

```diff
@@ -13,7 +13,7 @@ namespace mbgl {
 class Projection {
 public:
     // Map pixel width at given scale.
-    static double worldSize(double scale) {
+    static constexpr double worldSize(double scale) {
```

Comment:
> using `constexpr` allows clearer understanding of what we want to do

How so? We don't inherently intend that these calculations or constructors happen at compile time, and the fact that they (currently) _can_ is not an essential feature of the code.

I don't think we should go around adding `constexpr` everywhere it's possible to do so. If we're talking about introducing coding guidelines for C++ features, then I would say my guideline for `constexpr` is "use only in cases where compile-time evaluation is an intended or essential feature of the code, or for general-purpose library functions which may need to be used in `constexpr` contexts."